### DISCUSSION TITLE
Comparison of #multi-origin prefix vs RPKI

### DISCUSSION THREAD

**Discussion by SichangHe** [SichangHe/internet_route_verification#175] [General]
> Section 4 also mentions that "prefixes with multiple origins in route objects is
40X larger than" in BGP. OK, 40x is surprisingly high but I wonder how this
number compares to RPKI since ASes plan several origins (for example
a DDoS scrubber service) for fast response although it is not active in BGP.

This seems to be the last TODO in #138, right? @cunha.

**Comment by cunha** [comment]
I don't think RPKI will have many additional ASes. Might actually be *less* than in BGP (contrary to what the reviewer is mentioning). I agree we should give this low priority compared to other issues.
